application/run.sh

```diff
@@ -31,6 +31,11 @@ if [[ $rc != 0 ]]; then
   exit $rc;
 fi
 
+if [[ $APP_ENV == "dev" ]]; then
+    export XDEBUG_CONFIG="remote_enable=1 remote_host="$REMOTE_DEBUG_IP
```

Comment:
Does there need to be some kind of concatenation operator before the `$REMOTE_DEBUG_IP`? Or perhaps putting it into the string? (something like `export XDEBUG_CONFIG="remote_enable=1 remote_host=${REMOTE_DEBUG_IP}"`?)

I'm by no means an expert on shell script syntax. I was just surprised by the way it currently is written.